### DISCUSSION TITLE
Fix dependabot errors on deleted directory

### DISCUSSION THREAD

**Discussion by kurmasz** [community/community#34129] [Pull Requests]
I deleted some directories from a repository I haven't touched since March 2020, and now I'm getting a lot of dependabot errors with this message:

`Dependabot tried to update this pull request, but something went wrong. We're looking into it, but in the meantime you can retry the update by commenting @dependabot rebase.`

The issue below from 2021 suggests that I need to ask for a manual "re-detection"; but, I can't figure out how to get past the "knowledge base" menus so I can actually create a ticket.

https://github.com/dependabot/dependabot-core/issues/2041

Is there now a way to run this re-detection myself?  If not, how do I get to a place where I can actually submit a ticket?

(For what it's worth, the repo is:  https://github.com/kurmasz-SampleCode/CIS371-SampleCode

**Comment by airtower-luna** [comment]
>  If not, how do I get to a place where I can actually submit a ticket?

Go to https://support.github.com/, and click "my tickets" in the top right. On that site there's a green "new ticket" button. Good luck!
